### PR TITLE
Update DWS example to contain maxRunDurationSeconds annotation in the Job definition

### PR DESCRIPTION
**Pull request by PBundyra**
Update DWS example to contain maxRunDurationSeconds annotation in the Job definition

### CODE REVIEW

File: tutorials-and-examples/workflow-orchestration/dws-examples/job.yaml [hunk 1]
```diff
@@ -5,6 +5,8 @@ metadata:
   namespace: default
   labels:
     kueue.x-k8s.io/queue-name: dws-local-queue
+  annotations:
+    provreq.kueue.x-k8s.io/maxRunDurationSeconds : "600"
```
Comment (alculquicondor) [inline tutorials-and-examples/workflow-orchestration/dws-examples/job.yaml:9]:
```suggestion
    provreq.kueue.x-k8s.io/maxRunDurationSeconds: "600"
```

Comment (alculquicondor) [reply tutorials-and-examples/workflow-orchestration/dws-examples/job.yaml:9]: Also update the README to explain this field.

File: tutorials-and-examples/workflow-orchestration/dws-examples/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ available [here](https://cloud.google.com/kubernetes-engine/docs/how-to/provisio
 
 Files included:
 
-* kueue-manifests.yaml - [Kueue](https://kueue.sigs.k8s.io/) configuration files with ProvisioningRequest and DWS support enabled.
-* dws-queue.yaml - Kueue's Cluster and Local queues with ProvisioningRequest and DWS support enabled.
-* job.yaml - Sample job that requires GPU and uses DWS-enabled queue.
+* `kueue-manifests.yaml` - [Kueue](https://kueue.sigs.k8s.io/) configuration files with ProvisioningRequest and DWS support enabled.
+* `dws-queue.yaml` - Kueue's Cluster and Local queues with ProvisioningRequest and DWS support enabled.
+* `job.yaml` - Sample job that requires GPU and uses DWS-enabled queue. Contains annotation ` provreq.kueue.x-k8s.io/maxRunDurationSeconds` which sets `maxRunDurationSeconds` for the created ProvisioningRequest
```
Comment (alculquicondor) [inline tutorials-and-examples/workflow-orchestration/dws-examples/README.md:10]:
```suggestion
* `job.yaml` - Sample job that requires GPU and uses DWS-enabled queue. Contains optional annotation ` provreq.kueue.x-k8s.io/maxRunDurationSeconds` which sets `maxRunDurationSeconds` for the created ProvisioningRequest
```

**Review by ahg-g — APPROVED** [review]

**Pull request merged**